kindle_highlights.py

```diff
@@ -0,0 +1,54 @@
+import math
+import random
+
+MIN_HIGHLIGHTS = 5
+NUM_BOOKS_TO_SAMPLE = 3
+NUM_HIGHLIGHTS_TO_SAMPLE = 2
+
+def parse_highlights(highlights_json):
+    books_to_highlights = {}
+    for book in highlights_json["books"]:
+        # Ignore books that have fewer than MIN_HIGHTLIGHTS highlights
```

Comment:
Do we always want to ignore in this case?

Reply:
Yes

Reply:
We can change it but I find that that books with fewer than 5 highlights tend to be books I just read the sample. Usually its the same quote every time too making these random selections less useful.